core/dnsserver/register_test.go

```diff
@@ -29,3 +29,80 @@ func TestHandlers(t *testing.T) {
 		t.Errorf("Expected [testPlugin] from Handlers, got %v", hs)
 	}
 }
+
+func TestGroupingServers(t *testing.T) {
+	for i, test := range []struct {
+		configs []*Config
+		gpes    []string
```

Comment:
what does `gpes` stand for?

Reply:
oops... that stands for 'group' .. (in French .. groupe') - will change